tests/API/XCCDF/unittests/unit_helper.h

```diff
@@ -23,8 +23,8 @@
 #ifndef UNIT_HELPER_H_
 #define UNIT_HELPER_H_
 
-#include <xccdf_benchmark.h>
-#include <xccdf_policy.h>
+#include <XCCDF/xccdf_benchmark.h>
+#include <XCCDF_POLICY/xccdf_policy.h>
```

Comment:
this is very suspicious. if the include paths are setup correctly this shouldn't be necessary.